src/smoltcp_phy.rs

```diff
@@ -10,7 +10,10 @@ impl<'a, 'rx, 'tx, 'b> Device<'a> for &'b mut Eth<'rx, 'tx> {
     type TxToken = EthTxToken<'a>;
 
     fn capabilities(&self) -> DeviceCapabilities {
-        DeviceCapabilities::default()
+        let mut caps = DeviceCapabilities::default();
+        caps.max_transmission_unit = 1522;
```

Comment:
Maybe use the constant declared in `lib.rs` ?

Reply:
good call, I missed that constant.